packages/react-sdk/package.json

```diff
@@ -7,8 +7,8 @@
   "scripts": {
     "dev": "tsup src/index.ts --format esm,cjs --watch --out-dir=lib",
     "build": "yarn build:mdn-data && yarn build:args && yarn build:lib",
-    "build:mdn-data": "node --experimental-loader esbuild-node-loader ./bin/mdn-data.ts ./src/css",
-    "build:args": "node --experimental-loader esbuild-node-loader ./bin/generate-arg-types.ts './src/components/*.tsx !./src/**/*.stories.tsx !./src/**/*.ws.tsx'",
+    "build:mdn-data": "tsx ./bin/mdn-data.ts ./src/css",
+    "build:args": "tsx ./bin/generate-arg-types.ts './src/components/*.tsx !./src/**/*.stories.tsx !./src/**/*.ws.tsx'",
```

Comment:
I am fine moving from  esbuild-node-loader  to tsx if it solves the problem same fast but better, but do you understand the underlying problem it is fixing for us?

also cc @rpominov since you started using  esbuild-node-loader and we are now migrating to tsx

Reply:
@andrasbacsai in this case  esbuild-node-loader can be removed from deps, right? I would also check other usages of it and replace everywhere

Reply:
In case of @rpominov does not need it, I can remove it.

Reply:
I currently use `esbuild-register` for the migrations CLI in https://github.com/webstudio-is/webstudio-designer/pull/249

If there's reasons to use `txs` there, I can try to switch. But `esbuild-register` works fine for me, and I think it's faster than `txs`.

Also, maybe you should try replacing `node --experimental-loader esbuild-node-loader` with  `node -r esbuild-register`. I think this might be a better fix than switching to `tsx` if esbuild-register is indeed faster than tsx.

Reply:
tsx uses esbuild too, so there should be no perf difference, but please try and let us know, if there is, then we need to reconsider switching